EDU-18550 - Align internationalization docs with Catalog Translation API recommendation (#2752)
vtexdocs/dev-portal-content: docs/guides/vtex-io/Storefront-Guides/vtex-io-multi-language-stores/catalog-internationalization.md

Reviewer: Adicionar artigo definido ("the") antes do substantivo próprio ("Messages").
```markdown
>ℹ️ If your store uses catalog translation overrides via the Messages service (`vtex.messages`), you can check if a manual translation exists by running a query in the Messages GraphQL. Learn more in the section [Checking catalog message translations](https://developers.vtex.com/docs/guides/vtex-io-documentation-overwriting-the-messages-app#checking-catalog-message-translations) of the Overwriting the Messages app guide.
```

Reviewer: Capitalizar os nomes das seções (Category, Brand, Product e Specification values).
```markdown
5. Based on your scenario, check the sections ([Category](#category), [Brand](#brand), [Product](#product), [SKU](#sku), [SKU or product specification](#sku-or-product-specification), [Specification values](#specification-values)) for guidance on how to complete the main text box and the _Query variables_ section.
```

Reviewer: Mudar de voz passiva para ativa. Nomes de itens em menus em negrito.
```markdown
- `id`: Category ID. Every category in your store has a unique ID, which you can find under **Catalog > Categories** in the VTEX Admin.
```

Reviewer: Mudar de voz passiva para ativa. Nomes de itens em menus em negrito.
```markdown
- `id`: Brand ID. Every brand in your store has a unique ID that you can find under **Catalog > Brands** in the VTEX Admin.
```

Reviewer: Mudar de voz passiva para ativa. Nomes de itens em menus em negrito.
```markdown
- `id`: Product ID. Every product in your store has a unique ID, which you can find under **Catalog > Products and SKUs** in the VTEX Admin.
```

Reviewer: Mudar de voz passiva para ativa. Nomes de itens em menus em negrito.
```markdown
- `id`: SKU ID. Every product in your store has a unique ID, which you can find under **Catalog > Products and SKUs** in the VTEX Admin.
```

Reviewer: Espaço após ":" para padronizar o código.
```markdown
          "id": "91",
          "name": "Azul"
```

Reviewer: Atualizar links para não precisar de redirects.
```markdown
- **[Category](https://help.vtex.com/docs/tutorials/registering-a-category):** Name, keywords (similar words), page title (tag title), meta tag description, and URL slug (cross-border stores only).
- **[Brand](https://help.vtex.com/docs/tutorials/brand-registration-fields):** Name, keywords (similar words), page title (tag title), meta tag description, and URL slug (cross-border stores only).
- **[Product](https://help.vtex.com/docs/tutorials/adding-or-editing-products):** Name, keywords (similar words), page title (tag title), description, short description, meta tag description, and URL slug (cross-border stores only).
- **[SKU](https://help.vtex.com/docs/tutorials/adding-or-editing-skus):** Name.
- **[SKU or product specification](https://help.vtex.com/docs/tracks/specifications-concept-definition):** Name, description, and values.
```

Reviewer: Atualizar link do Rewriter app.
```markdown
> ℹ️ If you have a cross-border store, the `linkId` serves as the category URL slug. The [Rewriter](https://developers.vtex.com/docs/apps/vtex.rewriter) app will automatically create an alias using the translated slug for each target locale and store it in the `resolveAs` field for that locale's internal route. For example, a category with the slug `electronics` at `http://{storename}.com/us/electronics/d` could have the translated slug `eletronicos` for the `pt-BR` binding, as in `http://{storename}.com/br/eletronicos/d`.
```

Reviewer: Corrigir a linguagem do syntax highlight.
```markdown
```graphql
```

Reviewer: Corrigir a linguagem do syntax highlight.
```markdown
```graphql
```

Reviewer: Atualizar link do Rewriter app.
```markdown
> ℹ️ If you have a cross-border store, the `linkId` serves as the product URL slug. The [Rewriter](https://developers.vtex.com/docs/apps/vtex.rewriter) app will automatically create an alias using the translated slug for each target locale and store it in the `resolveAs` field for that locale's internal route. For example, a product with the slug `blue-top-retro-camera` at `http://{storename}.com/us/blue-top-retro-camera/p` could have the translated slug `camera-retro-azul` for the `pt-BR` binding, as in `http://{storename}.com/br/camera-retro-azul/p`.
```

Reviewer: Corrigir a linguagem do syntax highlight.
```markdown
```graphql
```

Reviewer: Corrigir a linguagem do syntax highlight.
```markdown
```graphql
```

Reviewer: Corrigir a linguagem de syntax highlight e a indentação.
```markdown
>```graphql
>query{
>   fieldValues(fieldId:"24"){
>     fieldValueId
>     value
>   }
>}
```

slug: "catalog-internationalization"
hidden: false
createdAt: "2020-08-31T17:18:54.238Z"
updatedAt: "2026-06-16T13:09:58.384Z"
---

> > ⚠️ This guide describes the GraphQL-based approach for translating catalog content using the `catalog-graphql` app. We recommend using the [Catalog Multi-Language API](https://developers.vtex.com/docs/guides/catalog-multi-language-integration-guide) instead, which is the latest solution for managing catalog translations. It offers granular control over translations for products, SKUs, categories, brands, and other entities, integrates natively with Intelligent Search, and supports Translation Management Systems (TMS). The simultaneous use of both the Catalog Multi-Language API and the GraphQL approach is not supported for catalog entities: once the Catalog Multi-Language feature is activated for your account, you will no longer be able to manage translations using GraphQL.

Reviewer: Mudar de voz passiva para ativa.
```suggestion
> ⚠️ This guide describes the GraphQL-based approach for translating catalog content using the `catalog-graphql` app. We recommend using the [Catalog Multi-Language API](https://developers.vtex.com/docs/guides/catalog-multi-language-integration-guide) instead, which is the latest solution for managing catalog translations. It offers granular control over translations for products, SKUs, categories, brands, and other entities, integrates natively with Intelligent Search, and supports Translation Management Systems (TMS). Catalog entities don't support using both the Catalog Multi-Language API and the GraphQL approach simultaneously. After activating the Catalog Multi-Language feature for your account, you'll no longer be able to manage translations using GraphQL.
```
